graph/cytoscape/cytoscape.go

```diff
@@ -279,6 +281,31 @@ func addTelemetry(ed *EdgeData, sn *tree.ServiceNode, nd *NodeData, o options.Ve
 	}
 }
 
+func addTLS(ed *EdgeData, sn *tree.ServiceNode, o options.VendorOptions) {
+	if sn.Parent == nil {
+		return
+	}
+
+	nmTLS, ok := sn.Metadata["hasmTLS"].(bool)
+	if !ok {
+		nmTLS = false
+	}
+
+	nMissingSideCars, ok := sn.Metadata["hasMissingSidecars"].(bool)
```

Comment:
Why the hasMissingSidecars logic is inside the addTLS ?
I think this was added before, isn't it more clear to add it in a separate addMissingSideCar() or a different name ?

Reply:
Here I am just reading the value of `hasMissingSidecars` added from a previous appender. I need that in order to determine whether the edge is using the mTLS or no.
AFAIK, appenders are designed to add information to nodes but not to edges. That is why I can add there a boolean telling that one all connections to that node are going to use mTLS.

I am not sure if that answer your question, did I?